### DISCUSSION TITLE
Mechanical applications

### DISCUSSION THREAD

**Discussion by ynedelec3** [vle-forge/irritator#25] [General]
Though I don't have present projects dedicated to a use of Irritator I explore potential mechanical applications and possible collaborations within that scope, which seems different to the domain covered by examples.

I produced a bouncing-ball model with qss1 (screenshot attached), as a prospective idea.
![ball](https://user-images.githubusercontent.com/22981923/110788816-67d62400-826f-11eb-9bfc-1e2e012496c1.jpg)

Within that scope and based upon bouncing masses, there is an interesting mathematical evaluation of Pi with a mechanical similitude, which could be interesting to reproduce and would allow to inspect model accuracy. Article is linked below.

If this seems of any interest...-)

[Playing_pool_with_p_the_number_p_from_a_billiard_p.pdf](https://github.com/vle-forge/irritator/files/6123001/Playing_pool_with_p_the_number_p_from_a_billiard_p.pdf)

**Comment by ynedelec3** [comment]
This model cannot stop the ball, it reaches a non-physical periodic regime at the end of the bounded total bouncing time (then the counter becomes linear with time as can be seen on figure).

**Comment by ynedelec3** [comment]
I had to multiply ball location by -1 to compare with threshold, the computation session launched by "Simulation run one"  automatically selects "Up detection" (a bug ?)

**Reply by quesnel** [reply]
I create an issue for the `Up detection` bug. Thx.

**Reply by ynedelec3** [reply]
Bad behaviour  of "Up detection" is fixed by this commit but negative multipliers remains necessary to simulate adequate bounces. I guess I misunderstand the role played by "Up detection"...
